yt/docker/ytsaurus/Dockerfile

```diff
@@ -27,16 +55,18 @@ RUN apt-get update && DEBIAN_FRONTEND=noninteractive TZ=Etc/UTC apt-get install
   libidn11-dev \
   lsb-release \
   lsof \
-  python3-pip \
-  python3.8  \
-  python3.8-distutils \
   python3.11 \
   python3.11-venv \
   strace \
   telnet \
   tini \
   unzip \
-  zstd
+  zstd \
+  google-perftools \
+  procps \
+  jq \
+  htop \
+  && rm -rf /var/lib/apt/lists/*
```

Comment:
It seems that the proper way is to call command `apt-cache clean` 

Also I suppose to split visually separate commands:
```
RUN apt-get update && \
    DEBIAN_FRONTEND=noninteractive TZ=Etc/UTC apt-get install -y \
        curl \
        dnsutils \
    ....
        htop && \
    apt-cache clean
```

The exact syntax is discussable

Reply:
Ok, I will try it.

Reply:
I did some googling, and I can't find the `apt-cache clean` command, but there is `apt-get clean`. However, it [does not](https://askubuntu.com/questions/1370546/what-is-the-difference-between-these-package-cache-clearing-methods) clean apt/lists, but it cleans caches. So it seems best to do both.